src/main/java/meditracker/command/HelpCommand.java

```diff
@@ -0,0 +1,41 @@
+package meditracker.command;
+
+import meditracker.ui.Ui;
+
+/**
+ * Represents a command to display help information for all available commands in MediTracker.
+ *
+ * This command lists all available commands along with a brief description of each.
+ * It extends the Command class and overrides the execute method to perform the action.
+ */
+public class HelpCommand extends Command {
+
+    /**
+     * Executes the help command by displaying a list of available commands and descriptions.
```

Comment:
You may want to consider changing "by displaying" to "to display".

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/210/commits/827dd72b58a9ea6835e368e7a8aa42dfb58b14ec. Thanks.